src-built-in/components/toolbar/src/dynamicToolbar.jsx

```diff
@@ -173,7 +179,7 @@ export default class Toolbar extends React.Component {
 		//console.log("Toolbar Render ");
 		if (!this.state.sections) return;
 		return (<FinsembleToolbar onDragStart={this.moveToolbar} onDragEnd={this.onPinDrag}>
-			<DragHandle />
+			{this.state.canDrag && <DragHandle />} {/*If this is not dockable, no need to show the drag handle */}
```

Comment:
@clausjs because down here I'd be asking myself, "what would cause canDrag to be false?"